test/e2e/const.go

```diff
@@ -84,9 +84,6 @@ var (
 	//go:embed data/rancher/azure-cluster.yaml
 	V2ProvAzureCluster []byte
 
-	//go:embed data/cluster-templates/docker-kubeadm.yaml
```

Comment:
Could you also remove the template?

Reply:
Template will be removed later on, when we update documentation. Currently removing template, it will break documentation